### ISSUE TITLE
DID registration fails on v.0.3.1

### ISSUE THREAD

**Issue by SebastianSchmidl** [digitalbazaar/did-client#11]
I wanted to test out the veres-one test network as described on your website: https://veres.one/developers/guides/create-did/

First of all the described procedure to install the client hasn't worked for me.
Running `npm install did-client` produced the following output:
```sh
npm ERR! code 1
npm ERR! Command failed: /usr/bin/git checkout implementation
npm ERR! error: pathspec 'implementation' did not match any file(s) known to git
npm ERR! 
```
After some research I found this repository and I hope, this is the right place to get a DID client for the veres one test network :slightly_smiling_face: 

Unfortunately, I was not able to register my own DID.
After changing the `did-veres-one`-dependency to point to the `master`-branch to get the project successfully installed (via `npm install`), the command `./did generate -r` returned a `TypeError` in the `veres.js`-script (see log output below).
I was not able to fix this issue myself.

## Environment:

- OS: Linux 4.18.0-10-generic x86_64 (Kubuntu 18.10)
- npm version: 6.4.1
- node.js version: 8.11.4
- git version: 2.19.1
- make: GNU Make 4.2.1
- cmake version: 3.12.1
- gcc: gcc (Ubuntu 8.2.0-7ubuntu1) 8.2.0

## Steps to reproduce

- Clone repository and checkout `v0.3.1` tag
- See following commands:
```sh
> /m/s/D/S/D/P/v/did-client > ⌂ v0.3.1 * > git status
HEAD detached at v0.3.1
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   package.json

no changes added to commit (use "git add" and/or "git commit -a")

> /m/s/D/S/D/P/v/did-client > ⌂ v0.3.1 * > git diff package.json
diff --git a/package.json b/package.json
index df3bbe7..6c77bab 100644
--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
   "dependencies": {
     "chalk": "^2.3.1",
     "did-client-veres-one": "veres-one/did-client-veres-one#implementation",
-    "did-veres-one": "veres-one/did-veres-one#implementation",
+    "did-veres-one": "veres-one/did-veres-one#master",
     "equihash-signature": "digitalbazaar/equihash-signature#master",
     "fs-extra": "^5.0.0",
     "get-stdin": "^5.0.1",

> /m/s/D/S/D/P/v/did-client > ⌂ v0.3.1 * > ./did generate --name "first" --description "First test DID generation" --register
[Veres One][test] Generating a new Veres One DID...
[Veres One][test] Generating keypair... (5-15 seconds)
[Veres One][test] DID: did:v1:test:nym:z279wo8Mnmw4QEcqnShPjzw5irCnAjqcUqcgXRGtydUdLtv2
[Veres One][test] Storing DID Document on disk...
[Veres One][test] DID Document stored in: <$HOME>/.testdid/did-v1-test-nym-z279wo8Mnmw4QEcqnShPjzw5irCnAjqcUqcgXRGtydUdLtv2.jsonld
[Veres One][test] Local DID generation successful.
[Veres One][test] Preparing to register a DID on Veres One...
[Veres One][test] Generating Equihash proof of work... (60-120 seconds)
TypeError: Cannot read property '0' of undefined
    at _send (/m/s/D/S/D/P/v/did-client/lib/drivers/veres.js:209:47)
    at <anonymous>

> /m/s/D/S/D/P/v/did-client > ⌂ v0.3.1 * > ./did list
did:v1:test:nym:z279wo8Mnmw4QEcqnShPjzw5irCnAjqcUqcgXRGtydUdLtv2

> /m/s/D/S/D/P/v/did-client > ⌂ v0.3.1 * > ./did register did:v1:test:nym:z279wo8Mnmw4QEcqnShPjzw5irCnAjqcUqcgXRGtydUdLtv2
[Veres One][test] Preparing to register a DID on Veres One...
[Veres One][test] Generating Equihash proof of work... (60-120 seconds)
TypeError: Cannot read property '0' of undefined
    at _send (/m/s/D/S/D/P/v/did-client/lib/drivers/veres.js:209:47)
    at <anonymous>
```

**Comment by mattcollier** [comment]
@CodeLionX Hi, thanks for giving this a try!  We definitely need to fix some dependencies here.

@dmitrizagidulin assigning this to you.

**Comment by dmitrizagidulin** [comment]
@CodeLionX Thanks again for reporting this. The fix with corrected dependencies has been published as `v0.3.2`. Please try installing again and let us know if you run into any issues.

**Comment by SebastianSchmidl** [comment]
@dmitrizagidulin Thank you very much for this quick fix! I'm now able to register my DID at the ledger. See log below:

```bash
> /m/s/D/S/D/P/v/did-client > ⌂ v0.3.1 * > ./did info did:v1:test:nym:FoJZaz3WY1EojWHykLgaZzscfAaZcn81jcQHpbfzvWGv                                         
{
  "@context": "https://w3id.org/veres-one/v1",
  "id": "did:v1:test:nym:FoJZaz3WY1EojWHykLgaZzscfAaZcn81jcQHpbfzvWGv",
  "authentication": [
    {
      "type": "Ed25519SignatureAuthentication2018",
      "publicKey": [
        {
          "id": "did:v1:test:nym:FoJZaz3WY1EojWHykLgaZzscfAaZcn81jcQHpbfzvWGv#authn-key-1",
          "type": "Ed25519VerificationKey2018",
          "owner": "did:v1:test:nym:FoJZaz3WY1EojWHykLgaZzscfAaZcn81jcQHpbfzvWGv",
          "publicKeyBase58": "FoJZaz3WY1EojWHykLgaZzscfAaZcn81jcQHpbfzvWGv"
        }
      ]
    }
  ],
  "grantCapability": [
    {
      "type": "Ed25519SignatureCapabilityAuthorization2018",
      "publicKey": [
        {
          "id": "did:v1:test:nym:FoJZaz3WY1EojWHykLgaZzscfAaZcn81jcQHpbfzvWGv#ocap-grant-key-1",
          "type": "Ed25519VerificationKey2018",
          "owner": "did:v1:test:nym:FoJZaz3WY1EojWHykLgaZzscfAaZcn81jcQHpbfzvWGv",
          "publicKeyBase58": "A2av9pGPBHYgSniAcyKrZ9t5N3LRSjxSQ8dyt4auaPzZ"
        }
      ]
    }
  ],
  "invokeCapability": [
    {
      "type": "Ed25519SignatureCapabilityAuthorization2018",
      "publicKey": [
        {
          "id": "did:v1:test:nym:FoJZaz3WY1EojWHykLgaZzscfAaZcn81jcQHpbfzvWGv#ocap-invoke-key-1",
          "type": "Ed25519VerificationKey2018",
          "owner": "did:v1:test:nym:FoJZaz3WY1EojWHykLgaZzscfAaZcn81jcQHpbfzvWGv",
          "publicKeyBase58": "QLPoYtWqQbTEzcXcGSd2mMfUGM2FhUJwKtRm6UZBQWh"
        }
      ]
    }
  ]
}
```
